test_vm/src/lib.rs

```diff
@@ -660,6 +675,27 @@ impl<'invocation, 'bs> InvocationCtx<'invocation, 'bs> {
         self.resolve_target(&self.msg.to).unwrap().1
     }
 
+    fn invoke_actor(&mut self) -> Result<Option<IpldBlock>, ActorError> {
+        std::panic::catch_unwind(std::panic::AssertUnwindSafe(|| self.invoke())).unwrap_or_else(
+            |panic| {
+                if self.actor_exit.borrow().is_some() {
+                    let exit = self.actor_exit.take().unwrap();
+                    if exit.code == 0 {
```

Comment:
This shouldn't happen in a panic, should it? It seems wrong, and I'd rather the test crashed than let this throuhg.